components/script/dom/globalscope.rs

```diff
@@ -2227,6 +2231,66 @@ impl GlobalScope {
         }))
     }
 
+    pub fn create_image_bitmap(
+        &self,
+        image: ImageBitmapSource,
+        options: &ImageBitmapOptions,
+    ) -> Rc<Promise> {
+        let global = self.global();
```

Comment:
We can use `self` anywhere we use `global` now instead, and remove this line.

Reply:
Done.